internal/lm/nvml.go

```diff
@@ -215,7 +215,8 @@ func isMPSCapable(manager resource.Manager) (bool, error) {
 func newGPUModeLabeler(devices []resource.Device) (Labeler, error) {
 	classes, err := getDeviceClasses(devices)
 	if err != nil {
-		return nil, err
+		klog.Warningf("Failed to create GPU mode labeler: failed to get device classes: %v", err)
```

Comment:
The one question is whether we should return an `nvidia.com/gpu.mode="unknown"` label here instead?

Reply:
I am not opposed. Do we have any precedent for using `unknown` for other labels where we are unable to gather the required information?

Reply:
We already return `unknown` when getting the mode below.

Reply:
Thanks, I clearly did not look at the code below carefully enough. I have updated this to return `unknown` instead.